### ISSUE TITLE
getting "TypeError: Cannot read properties of undefined (reading 'mochaOpts')" on WDIO v8

### ISSUE THREAD

**Issue by HananArgov** [LambdaTest/wdio-lambdatest-service#46]
we're using Jasmine as our runner, and at the end of the run we get the following error - stack trace:
```
[0-0] 2023-01-04T20:00:41.916Z ERROR @wdio/utils:shim: TypeError: Cannot read properties of undefined (reading 'mochaOpts')
[0-0]     at LambdaRestService.after (/Users/hananargov/Documents/mind-services/e2e-tests/node_modules/wdio-lambdatest-service/src/service.js:99:31)
[0-0]     at file:///Users/hananargov/Documents/mind-services/e2e-tests/node_modules/@wdio/utils/build/shim.js:31:27
[0-0]     at new Promise (<anonymous>)
[0-0]     at file:///Users/hananargov/Documents/mind-services/e2e-tests/node_modules/@wdio/utils/build/shim.js:28:47
[0-0]     at Array.map (<anonymous>)
[0-0]     at executeHooksWithArgsShim (file:///Users/hananargov/Documents/mind-services/e2e-tests/node_modules/@wdio/utils/build/shim.js:28:33)
[0-0]     at JasmineAdapter.run (file:///Users/hananargov/Documents/mind-services/e2e-tests/node_modules/@wdio/jasmine-framework/build/index.js:196:15)
[0-0]     at processTicksAndRejections (node:internal/process/task_queues:95:5)
[0-0]     at async Runner.run (file:///Users/hananargov/Documents/mind-services/e2e-tests/node_modules/@wdio/runner/build/index.js:145:24)
```

https://github.com/LambdaTest/wdio-lambdatest-service/blob/8df6716d68d8da3217ff9c1105153aaa8b0f4b2b/src/service.js#L99

**Comment by HananArgov** [comment]
i've changed the line to use `options` instead of `config` and no more exception on this:
```
    if (global.browser.options.mochaOpts && global.browser.options.mochaOpts?.bail && Boolean(result)) {
```

**Comment by piotrblasiak** [comment]
Any plans for a release with this fix? It's a blocker for us.

**Comment by HananArgov** [comment]
seems fixed in 1.0.15

**Comment by piotrblasiak** [comment]
Yes, the lambdatest support team were quick to respond to and fix this. They did forget however it seems to update the deps in the package.json but it works if you use --legacy-peer-deps
